src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -732,7 +748,7 @@ where
             input_challenges: assigned_challanges_instance,
             folded_X0: assigned_X0,
             folded_X1: assigned_X1,
-            input_W_commitment: assigned_instance_W_commitment_coordinates,
+            input_W_commitments: vec![assigned_instance_W_commitment_coordinates],
```

Comment:
Look at comment above and use all `W` here

Reply:
yeah, fixed it based on above comment